testcafe/tests/photos/photos_crud.js

```diff
@@ -63,14 +97,17 @@ test('Open Last pic', async t => {
   console.groupEnd()
 })
 
-test('Open a random pic (not first nor last)', async t => {
-  console.group('↳ ℹ️  Open a random pic (not first nor last)')
+test(`${TEST_VIEWER_OTHER}`, async t => {
+  console.group(`↳ ℹ️  ${FEATURE_PREFIX} : ${TEST_VIEWER_OTHER}`)
   //Both arrows show up. Navigatio to other pics is OK, Closing pic (X or 'esc') is Ok
   // We need at least 3 pics in our cozy for this test to pass
   const photoIndex = random(1, t.ctx.totalFilesCount - 2)
 
   console.log('Open random pic  > photoIndex ' + photoIndex)
   await photoViewer.openPhotoFullscreen(photoIndex)
+  await t.fixtureCtx.vr.takeScreenshotAndUpload(
```

Comment:
you could put `t.fixtureCtx.vr.takeScreenshotAndUpload` in a const to ease readability

Reply:
I don't think so. `t.fixutureCtx` is needed to get the fixture and test context 
https://devexpress.github.io/testcafe/documentation/test-api/test-code-structure.html#sharing-variables-between-fixture-hooks-and-test-code